### DISCUSSION TITLE
Is there a way to just allow leaf nodes to be selectable

### DISCUSSION THREAD

**Discussion by abubakkardh** [dowjones/react-dropdown-tree-select#593] [Q&A]
If you have a general query about the usage of react-dropdown-tree-select, or if you're not sure how to accomplish a particular task using the component; please raise a question on [stackoverflow.com](https://stackoverflow.com) with [react-dropdown-tree-select](https://stackoverflow.com/questions/tagged/react-dropdown-tree-select) tag.

If you think you found a bug or have a feature request, please use the `Bug Report`/`Feature request` template respectively.

**Comment by gandhis1** [comment]
When passing the `data` object to the component, set the `disabled` flag on non-leaf nodes. You can also accomplish the same through CSS by selecting for any `node` which does not have the `node-leaf`  class, and hiding the check box.
